lua/nvim-treesitter/indent.lua

```diff
@@ -242,7 +242,7 @@ function M.get_indent(lnum)
           end
         else
           -- aligned indent
-          if c_delim_node and c_is_last_in_line and c_srow and o_srow ~= c_srow and c_srow < lnum - 1 then
+          if c_is_last_in_line and o_srow ~= c_srow and c_srow < lnum - 1 then
```

Comment:
if c_is_last_in_line is true, then c_srow and c_delim_node must also exist, so checking for all three is unncessary